### PR TITLE
Clean spec file (mainly: use %license).

### PR DESCRIPTION
**Pull request by DimStar77**
Version not bumped, as a SR to TW for just this is not needed for now)

### CODE REVIEW

File: package/yast2-alternatives.spec [hunk 1]
```diff
@@ -15,23 +15,22 @@
 # Please submit bugfixes or comments via https://bugs.opensuse.org/
 #
 
+
 Name:           yast2-alternatives
 Version:        4.2.0
```
Comment (lslezak) [inline package/yast2-alternatives.spec]: Please bump the version to 4.2.1 so the change is automatically submitted to Factory by Jenkins.

**Comment by DimStar77** [comment]
version bumped, as requested

**Review by jreidinger — APPROVED** [review]

**Comment by lslezak** [comment]
Could you please rebase the changes and resolve the conflicts? Thank you!

**Comment by DimStar77** [comment]
I rebased this once again - sorry for the delay

**Pull request merged**